### ISSUE TITLE
is this normal?

### ISSUE THREAD

**Issue by emerymastering** [apohl79/audiogridder#161]
Hello I get this screen once starting server.
![Screen Shot 2020-10-04 at 13 55 33](https://user-images.githubusercontent.com/66089231/95014768-5c8c2600-0649-11eb-8657-5ffd887757e3.png)
In Accessibility I have it all enabled. So wonder if it should be warning me every time or not?

**Comment by apohl79** [comment]
Remove it and add it again should fix the issue.

**Comment by emerymastering** [comment]
yess this did the trick, thanks:)
